lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -248,6 +284,15 @@ class RenderPopover extends RenderShiftedBox {
 
     final borderPath = _buildBorderPath(direction, arrowCenter);
 
+    if (elevation != 0.0) {
+      context.canvas.drawShadow(
+        borderPath,
+        _shadowColor,
+        _elevation,
+        _backgroundColor.alpha != 0xFF,
```

Comment:
what's this condition?

Reply:
This indicates whether the background is not opaque.

The documentation says `The transparentOccluder argument should be true if the occluding object is not opaque.`

Reply:
Ok. Let's make this abundantly clear. Let's define a `final` variable up above the `drawShadow()` call which is called `isMenuTranslucent`:

```dart
final isMenuTranslucent = _backgroundColor.alpha != 0xFF;
context.canvas.drawShadow(
  //...
  isMenuTranslucent,
);
```